<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>充电机测试</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>数据位：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_DataBits">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>串口状态：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="checkBox_ShowInHex">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>串口：</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_ComState">
          <property name="text">
           <string>关闭</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>波特率：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_Com"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_BoundRate">
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_OnOff">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBox_ParityBits">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="pushButton_Clear">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>停止位：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboBox_StopBits">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>校验位：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_12"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_13"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_14"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_15"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_16"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_17"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_18"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_19"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineEdit_20"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_21"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_22"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_23"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_24"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_25"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_26"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_27"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_28"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_29"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineEdit_30"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_31"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_32"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_33"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_34"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_35"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_36"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_37"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_38"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_46">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_39"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineEdit_40"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
